rel/config.exs

```diff
@@ -0,0 +1,43 @@
+# Import all plugins from `rel/plugins`
+# They can then be used by adding `plugin MyPlugin` to
+# either an environment, or release definition, where
+# `MyPlugin` is the name of the plugin module.
+if false do
+  Path.join(["rel", "plugins", "*.exs"])
+  |> Path.wildcard()
+  |> Enum.map(&Code.eval_file(&1))
+end
+
+use Mix.Releases.Config,
+    default_release: :default,
+    default_environment: Mix.env()
+
+environment :dev do
+  # If you are running Phoenix, you should make sure that
+  # server: true is set and the code reloader is disabled,
+  # even in dev mode.
```

Comment:
We have code reloader enabled in dev mode.
I'm guessing we don't care a lot about releases in dev mode, but I thought I should point it out anyway

Reply:
Yeah, we don't want to make dev releases. I'd be down to take this out altogether.

Thoughts?